en/SUMMARY.md

```diff
@@ -1,152 +1,144 @@
-      * [struct GroundTruth](api_reference/structmavsdk_1_1_telemetry_1_1_ground_truth.md)
-      * [struct Health](api_reference/structmavsdk_1_1_telemetry_1_1_health.md)
-      * [struct Imu](api_reference/structmavsdk_1_1_telemetry_1_1_imu.md)
-      * [struct MagneticFieldFrd](api_reference/structmavsdk_1_1_telemetry_1_1_magnetic_field_frd.md)
-      * [struct Odometry](api_reference/structmavsdk_1_1_telemetry_1_1_odometry.md)
-      * [struct Position](api_reference/structmavsdk_1_1_telemetry_1_1_position.md)
-      * [struct PositionBody](api_reference/structmavsdk_1_1_telemetry_1_1_position_body.md)
-      * [struct PositionNed](api_reference/structmavsdk_1_1_telemetry_1_1_position_ned.md)
-      * [struct VelocityNed](api_reference/structmavsdk_1_1_telemetry_1_1_velocity_ned.md)
-      * [struct VelocityBody](api_reference/structmavsdk_1_1_telemetry_1_1_velocity_body.md)
-      * [struct PositionVelocityNed](api_reference/structmavsdk_1_1_telemetry_1_1_position_velocity_ned.md)
-      * [struct Quaternion](api_reference/structmavsdk_1_1_telemetry_1_1_quaternion.md)
-      * [struct RcStatus](api_reference/structmavsdk_1_1_telemetry_1_1_rc_status.md)
-      * [struct StatusText](api_reference/structmavsdk_1_1_telemetry_1_1_status_text.md)
-    * [class Tune](api_reference/classmavsdk_1_1_tune.md)
-      * [struct TuneDescription](api_reference/structmavsdk_1_1_tune_1_1_tune_description.md)
-    * [namespace mavsdk (globals)](api_reference/namespacemavsdk.md)
+  * [QuickStart](cpp/quickstart.md)
+  * [Guide](cpp/guide/README.md)
+    * [Usage/Paradigms](cpp/guide/general_usage.md)
+    * [Connecting to Systems (Vehicles)](cpp/guide/connections.md)
+    * [Managing Systems](cpp/guide/using_plugins.md)
+    * [System Information](cpp/guide/system_information.md)
+    * [Telemetry](cpp/guide/telemetry.md)
+    * [Actions - Take Off, Arm, ...](cpp/guide/taking_off_landing.md)
+    * [Missions](cpp/guide/missions.md)
+    * [Offboard Control](cpp/guide/offboard.md)
+    * [Follow Me](cpp/guide/follow_me.md)
+    * [VTOL Support](cpp/guide/vtol.md)
+    * [Building C++ Apps](cpp/guide/toolchain.md)
+    * [Plugin/Test Logging](cpp/guide/dev_logging.md)
+    * [Testing](cpp/guide/test.md)
+    * [Building library from Source](cpp/guide/build.md)
+    * [Building mavsdk_server from Source](cpp/guide/build_mavsdk_server.md)
+    * [Generate API docs from Source](cpp/guide/build_docs.md)
+    * [Coding Style](cpp/guide/code_style.md)
+  * [Examples](cpp/examples/README.md)
+    * [Takeoff and Land](cpp/examples/takeoff_and_land.md)
+    * [Fly Mission](cpp/examples/fly_mission.md)
+    * [Fly QGC Plan Mission](cpp/examples/fly_mission_qgc_plan.md)
+    * [Offboard Mode - Velocity Control](cpp/examples/offboard_velocity.md)
+    * [Follow Me Mode](cpp/examples/follow_me.md)
+    * [VTOL Transitions](cpp/examples/transition_vtol_fixed_wing.md)
+  * [API Changes](cpp/api_changes.md)
```

Comment:
I wonder if this should below troubleshooting in the hierarchy. Also does it only apply to C++?

Reply:
It is only C++, so far.